CanHazFunny/CanHazFunny.Tests/JesterTests.cs

```diff
@@ -1,10 +1,68 @@
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-
 namespace CanHazFunny.Tests
 {
+    using System;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Moq;
+
     [TestClass]
     public class JesterTests
     {
-        
+        [TestMethod]
+        public void TellJoke_WhenCalled_RunsBothInterfacedMethods()
+        {
+            Mock<IJokeService> iJokeService = new ();
+            Mock<IOutputJoke> iOutputJoke = new ();
```

Comment:
In general we don't include the "i" when naming variables
```suggestion
            Mock<IJokeService> jokeService = new ();
            Mock<IOutputJoke> outputJoke = new ();
```